EventBot/Modules/EventModule.cs

```diff
@@ -339,14 +339,14 @@ public EventManagementModule(EventManagementService events, DatabaseService data
             }
 
             [Command("open")]
-            [Summary("Open registration for event here.")]
+            [Summary("Open registration for event here. Warning: Ensure everything is correct before doing this.")]
```

Comment:
Don't. everything is correctable. You can always close event and add new roles.
```suggestion
            [Summary("Open registration for event here.")]
```